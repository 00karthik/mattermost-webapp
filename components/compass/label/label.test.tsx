// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.

import React from 'react';
import {shallow} from 'enzyme';

import Label from './label';

describe('components/compass/Label', () => {
    test('should match snapshot', () => {
        const Props = {
            children: 'text',
            size: 'small',
        };

        const wrapper = shallow(
            <Label
                {...Props}
            />,
        );

        expect(wrapper).toMatchSnapshot();
        expect(wrapper.find('span').props().className).toBe('Label Label___small');
    });
});