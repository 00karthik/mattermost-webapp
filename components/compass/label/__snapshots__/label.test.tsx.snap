// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/compass/Label should match snapshot 1`] = `
<span
  className="Label Label___small"
>
  text
</span>
`;
